### DISCUSSION TITLE
Where I can find more Dicom files for the import?

### DISCUSSION THREAD

**Discussion by SirinC** [mlavik1/UnityVolumeRendering#258] [Q&A] [answered]
Where I can find more Dicom files for the import?

**Comment by mlavik1** [comment, marked as answer]
Hi! If you're looking for free sample DICOM files, you can have a look at this list: https://github.com/mlavik1/UnityVolumeRendering/wiki/Sample-datasets

The "Cancer Imaging Archive" has *a lot* of datasets.
